src/worker.rs

```diff
@@ -151,13 +152,20 @@ fn list_files(logger: &Logger, name: &str, recursive: bool) -> Result<Vec<File>,
                                         "Parsed file";
                                         "name" => name.clone(), "size" => size, "revision" => revision, "author" => author.clone(), "date" => date.to_string(),
                                     );
-                                    files.push(File {
-                                        path: name,
-                                        size,
-                                        revision,
-                                        last_author: author,
-                                        last_changed: date,
-                                    });
+                                    let sha_output =
+                                        Command::new("git svn find-rev").arg(format!("{}", revision)).output();
```

Comment:
The form that takes individual tokens is better imo. Requires less parsing. Also, the format! call is unnecessary. Should be possible to just call `to_string` or something?